eradiate/radprops/rad_profile.py

```diff
@@ -613,8 +734,8 @@ def to_dataset(self, spectral_ctx=None):
         profile = self.eval_thermoprops_profile()
         return make_dataset(
             wavelength=spectral_ctx.wavelength,
-            z_level=profile.z_level.values,
-            z_layer=profile.z_layer.values,
+            z_level=to_quantity(profile.z_level),
+            z_layer=to_quantity(profile.z_layer),
```

Comment:
I think you're missing a conversion to unitless here (this should generate a warning).

Reply:
I am not sure if I understand.
The `z_level` and `z_layer` coordinates in the thermophysical `profile` object will have length units. I tried again, I don't see a warning.

Reply:
If `make_dataset()` interprets units, then forget what I wrote. I just raise that point because `profile.z_level.values` is unitless while `to_quantity(profile.z_level)` is not.

Reply:
Ah, you were right that something is wrong; I found a mistake in the decorator of `radprops.rad_profile.make_dataset()`. I think everything is fine now.